input/reproSchemaScript/reprotofhirjson.py

```diff
@@ -0,0 +1,282 @@
+
+    schema = open(questionnaire + "/"+ questionnaire+"_schema", "r")
+    contents = schema.read()
+    j = json.loads(contents)
+    schema.close()
+
+    fhir_questionnaire["resourceType"] = "Questionnaire"
+    fhir_questionnaire["meta"]= {
+    "profile" : [
+      "https://voicecollab.ai/fhir/StructureDefinition/vbai-questionnaire"
+    ]
+    }
+    fhir_questionnaire["id"] = j["@id"].replace('_', '')
+    fhir_questionnaire["url"] = QUESTIONNAIRE_URI + j["@id"].replace('_', '')
+    fhir_questionnaire["title"] =  j["@id"]
+
+    fhir_questionnaire["text"] =  {"status" : "generated","div" : "<div xmlns=\"http://www.w3.org/1999/xhtml\">Placeholder</div>"}
+    fhir_questionnaire["version"] = "1.4.0"
+    fhir_questionnaire["status"] = "active"
+    fhir_questionnaire["date"] = "2023-11-20T11:33:15-05:00"
+    fhir_questionnaire["publisher"] = "KinD Lab"
+    fhir_questionnaire["contact"] =  [
+    {
+      "name" : "KinD Lab",
+      "telecom" : [
+        {
+          "system" : "url",
+          "value" : "http://fhir.kindlab.sickkids.ca"
+        }
+      ]
+    }
+  ]
+
+    fhir_questionnaire["item"] = []
+    group = dict()
+    # default description
+
+    group["linkId"] = "T1"
+
+    if "preamble" in j.keys():
+        if isinstance(j["preamble"], dict):
+            group["text"] = j["preamble"]["en"]
+        elif isinstance(j["preamble"], str):
+            group["text"]= j["preamble"]
+    else:
+        group["text"] = ""
+    
+    group["type"] = "group"
+    group["item"] = []
+
+
+    questions = j["ui"]["visibility"]
+
+
+    # iterate over every questions
+    for i in questions:
+        curr_question = dict()
+        question_file = open(questionnaire +"/items/"+i["variableName"], "r")
+        question_contents = question_file.read()
+
+        question_json = json.loads(question_contents)
+        question_file.close()
+
+        curr_question["linkId"] =  i["variableName"]
+        # default assigned type
+        item_type = "string"
+        # adding questions type
+        if "inputType" in question_json["ui"]:
+            if question_json["ui"]["inputType"] == "radio":
+                item_type = "choice"
+            elif question_json["ui"]["inputType"] in ("number", "xsd:int"):
+                item_type = "integer"
+
+        curr_question["type"] = item_type
+        # adding questions text
+        if "question" in question_json and isinstance(question_json["question"], dict):
+            curr_question["text"] = str(question_json["question"]["en"])
+        else:           
+            curr_question["text"] = str(question_json["prefLabel"])
+
+        # add options
+        if "responseOptions" in question_json:
+            curr_question = add_options(curr_question, question_json, questionnaire, i["variableName"],  mode)
+        group["item"].append(curr_question)
+
+    fhir_questionnaire["item"].append(group)
+  
+
+if __name__ == '__main__':
+    convert_to_fsh(sys.argv[1], sys.argv[2] )
```

Comment:
use argparse for the input arguments: https://docs.python.org/3/library/argparse.html